src/OrchardCore.Modules/OrchardCore.Forms/Views/Items/SelectPart.cshtml

```diff
@@ -14,14 +14,14 @@
         fieldValue = fieldEntry.AttemptedValue;
         if (fieldEntry.Errors.Count > 0)
         {
-            fieldClass = "form-control input-validation-error";
+            fieldClass += " input-validation-error";
```

Comment:
string.Format or $

Reply:
Why would that be better? This concatenation is simple, the intent is clear, and it's also shorter than writing the following (and especially using `string.Format()`:

```c#
fieldClass = $"{fieldClass} input-validation-error";
```

Reply:
Yeah. Not sure either. Unless there is a convincing reason I would rather leave it as is

Reply:
I think the comment is about injecting the extra content when the initial value is created, in a single line. Either way.